### ISSUE TITLE
Issue with display: grid for height.

### ISSUE THREAD

**Issue by VicFrolov** [FormidableLabs/nuka-carousel#522]
# Bugs and Questions

### Describe Your Environment

- What version of nuka-carousel are you using? 4.5.3
- What version of React are you using? 16.8.5
- What browser are you using? Chrome
- What machine are you on? OSX

### Describe the Problem
The parent container uses grids in order to determine the dimensions of the images. I was able to get it to respond correctly based on the width by seeing this closed issue: https://github.com/FormidableLabs/nuka-carousel/issues/440, and applying the `gridTemplateColumns: 'minmax(0, 1fr')`. However, on render, the height is too small, but if I click on any of the arrows, or if I resize the browser size, the height then becomes correct.

```
    <div
      style={{
        display: 'grid',
        gridTemplateColumns: 'minmax(0, 1fr)',
      }}
    >
      <Carousel>
        {images.map(image => (
          <img
            alt={image.altText}
            key={image.src}
            src={image.src}
          />
        ))}
      </Carousel>
    </div>
```
**Expected behavior:** 
The contents to take their respective widths/height.

**Actual behavior:** 
 Height is about half of what it should be, and then on any re-render, it becomes the correct height.

**Comment by dispix** [comment]
I'm having a similar issue but my parent node is in `display: flex` and the height of the carousel is actually *twice*  the appropriate height. Just like you, it adapts and uses the correct height once I scroll.

**Comment by VicFrolov** [comment]
Good catch on the issue number! Def seems related.

**Comment by sarmeyer** [comment]
Should be fixed with the release of 4.5.10 - give it a try once upgrading and let me know if you still experience the issue!
